src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -3485,6 +3486,41 @@ internal void SymbolDeclaredEvent(Symbol symbol)
             EventQueue?.TryEnqueue(new SymbolDeclaredCompilationEvent(this, symbol.GetPublicSymbol()));
         }
 
+        internal override void SerializePdbEmbeddedCompilationOptions(BlobBuilder builder)
+        {
+            WriteValue("checked", Options.CheckOverflow.ToString());
+            WriteValue("nullable", Options.NullableContextOptions.ToString());
+            WriteValue("unsafe", Options.AllowUnsafe.ToString());
+            WriteValue("language-version", LanguageVersion.ToString());
+            WriteValue("optimization", Options.OptimizationLevel.ToString());
```

Comment:
>Options.OptimizationLevel.ToString() [](start = 39, length = 36)

I think we still need some custom serialization here to include `debug+` ("release", "debug", "debug+") - I filed an issue to add `debug+` to OptimizationLevel enum, but that will probably not happen soon enough: [https://github.com/dotnet/roslyn/issues/44809](https://github.com/dotnet/roslyn/issues/44809)

It's also better to serialize enums explicitly. ToString is doing Reflection, which is inefficient.